docs/snippets/Snippets.NUnit/Snippets.NUnit.csproj

```diff
@@ -10,7 +10,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.7.2" />
-    <PackageReference Include="NUnit" Version="3.13.3" />
+    <PackageReference Include="NUnit" Version="4.0.0-dev-07984" />
```

Comment:
⚠️ Do we really want to update the main docs reference with this early of a version? Or should this doc update perhaps be "ready and waiting" here in this PR and merged as soon as v4.0.0 is released? I don't necessarily feel it has to be that way but I want us to consider it.

Reply:
We should change this now to beta.1. Ill do that

Reply:
@OsirisTerje yes I was referring to any pre-release (alpha / beta). But that's your call and I'll respect it. Wanted to note it.

Reply:
Sure :-)  I just changed it to beta.1 though.  We'll keep it live so it matches the most recent version.